src/App.tsx

```diff
@@ -20,12 +30,15 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                onChange={(ev) => {
+                  setQuery(ev.target.value);
+                }}
```

Comment:
you can optimize it and move function to variable;
So
const handleChangeQuery = (event) => setQuery(event.target.value)
onChange={handleChangeQuery}

Reply:
Then i become an error
<img width="634" alt="image" src="https://user-images.githubusercontent.com/106486742/200124640-d1757ead-7b33-4b39-bf77-e8083434ab46.png">

Reply:
You need to specify type for an event. For example, React.ChangeEvent\<HTMLInputElement\>.